app/javascript/packs/managers/cuisines.js

```diff
@@ -84,6 +84,18 @@ $(function () {
         }
       });
       break;
+    case "new":
```

Comment:
失敗時の`$('body').data('action')`は`create`です。確認するとわかるかと思います。
なのでこうすれば今PRで崩れてる、と言うか非表示にしたいid=choiced_rawmaterialのdivタグはnewと同じようにできます。
要するに、条件でnewでもcreateでも同じことするようにしてください。

```suggestion
    case "create":
    case "new":
```